cristin-person/src/test/java/no/unit/nva/cristin/person/institution/fetch/FetchPersonInstitutionInfoHandlerTest.java

```diff
@@ -0,0 +1,180 @@
+            sendQuery(null, Map.of(PERSON_ID, INVALID_PATH_PARAM, ORG_ID, VALID_INSTITUTION_ID));
+
+        assertEquals(HttpURLConnection.HTTP_BAD_REQUEST, gatewayResponse.getStatusCode());
+        assertEquals(APPLICATION_PROBLEM_JSON.toString(), gatewayResponse.getHeaders().get(HttpHeaders.CONTENT_TYPE));
+        assertThat(gatewayResponse.getBody(), containsString(FetchPersonInstitutionInfoHandler.INVALID_PERSON_ID));
+    }
+
+    @Test
+    void shouldThrowBadRequestOnInvalidOrganizationPathParam() throws IOException {
+        GatewayResponse<PersonInstitutionInfo> gatewayResponse =
+            sendQuery(null, Map.of(PERSON_ID, VALID_PERSON_ID, ORG_ID, INVALID_PATH_PARAM));
+
+        assertEquals(HttpURLConnection.HTTP_BAD_REQUEST, gatewayResponse.getStatusCode());
+        assertEquals(APPLICATION_PROBLEM_JSON.toString(), gatewayResponse.getHeaders().get(HttpHeaders.CONTENT_TYPE));
+        assertThat(gatewayResponse.getBody(),
+            containsString(FetchPersonInstitutionInfoHandler.INVALID_ORGANIZATION_ID));
+    }
+
+    @Test
+    void shouldThrowBadRequestOnMissingPersonPathParam() throws IOException {
+        GatewayResponse<PersonInstitutionInfo> gatewayResponse =
+            sendQuery(null, Map.of(ORG_ID, VALID_INSTITUTION_ID));
+
+        assertEquals(HttpURLConnection.HTTP_BAD_REQUEST, gatewayResponse.getStatusCode());
+        assertEquals(APPLICATION_PROBLEM_JSON.toString(), gatewayResponse.getHeaders().get(HttpHeaders.CONTENT_TYPE));
+        assertThat(gatewayResponse.getBody(), containsString(INVALID_PERSON_ID));
+    }
+
+    @Test
+    void shouldThrowBadRequestOnMissingOrganizationPathParam() throws IOException {
+        GatewayResponse<PersonInstitutionInfo> gatewayResponse =
+            sendQuery(null, Map.of(PERSON_ID, VALID_PERSON_ID));
+
+        assertEquals(HttpURLConnection.HTTP_BAD_REQUEST, gatewayResponse.getStatusCode());
+        assertEquals(APPLICATION_PROBLEM_JSON.toString(), gatewayResponse.getHeaders().get(HttpHeaders.CONTENT_TYPE));
+        assertThat(gatewayResponse.getBody(), containsString(INVALID_ORGANIZATION_ID));
+    }
+
+    private URI getCristinUriFromIdentifiers() {
+        return URI.create(EXPECTED_CRISTIN_URI);
+    }
+
+    private GatewayResponse<PersonInstitutionInfo> sendQuery(Map<String, String> queryParams,
+                                                             Map<String, String> pathParam)
+        throws IOException {
+
+        InputStream input = requestWithParams(queryParams, pathParam);
+        handler.handleRequest(input, output, context);
+        return GatewayResponse.fromOutputStream(output);
+    }
+
+    private InputStream requestWithParams(Map<String, String> queryParams, Map<String, String> pathParams)
+        throws JsonProcessingException {
+
+        return new HandlerRequestBuilder<Void>(OBJECT_MAPPER)
+            .withBody(null)
+            .withQueryParameters(queryParams)
```

Comment:
queryParameters are never sent. Remove if not in use

Reply:
If the intention is to validated that no query parameters are being sent, this should be an explicit test.